tox.ini

```diff
@@ -1,28 +0,0 @@
-# Tox (https://tox.readthedocs.io/) is a tool for running tests
-# in multiple virtualenvs. This configuration file will run the
-# test suite on all supported python versions. To use it, "pip install tox"
-# and then run "tox" from this directory.
-# 
-# To run different environments, try `tox -e py27` or `tox -e docs`
-# 
-
-[tox]
-envlist = py36, py37, py38, py39
-
-[gh-actions]
-python =
-    3.6: py36
-    3.7: py37
-    3.8: py38
-    3.9: py39
-
-[testenv]
-commands =
-  pip install -e .
-  {envpython} setup.py test
-
-[testenv:docs]
-deps = -rdocs/requirements.txt
-commands =
-  mkdocs build
-  pdoc --html -o public/ fpdf
```

Comment:
to my mind, this is "self documenting", what do you think?

Reply:
It's also "self documenting" there then: https://github.com/alexanderankin/pyfpdf/blob/qpdf/.github/workflows/continuous-integration-workflow.yml#L44

I just though there were no point in duplicating this, given `tox` is not used anymore in the GitHub Actions pipeline